knative-operator/deploy/crds/operator_v1alpha1_knativekafka_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  name: example-knativekafka
+spec:
+  # Add fields here
+  size: 3
```

Comment:
Default stuff generated by the operator-sdk cli, gonna change it later or remove this example CR file.

Reply:
Let's just ditch it now anyway. It only confuses peeps and isn't even functional.

Reply:
agreed on removing